herbert_core/configuration/@config_store/config_store.m

```diff
@@ -293,8 +296,12 @@ function set_saveable(this,class_instance,is_it)
             % set up unit test directories. But this can not set up
             % internal private dependent fields so a configuration can not
             % have such fields! (the setting got lost)
+            
+            % Do we really need this? It seems works fine without it and
+            % this causes strange side effects
```

Comment:
What side effects?

Reply:
It causes recursive execution if one tries to look on variables under debugger. As I am not dealing with config_store here, I would avoid digging deeper within this ticket.